MekHQ/src/mekhq/campaign/stratcon/StratconRulesManager.java

```diff
@@ -1039,7 +1044,9 @@ public static boolean isUnitDeployedToStratCon(Unit u) {
         // this is a little inefficient, but probably there aren't too many active AtB
         // contracts at a time
         return u.getCampaign().getActiveAtBContracts().stream().
-            anyMatch(contract -> contract.getStratconCampaignState().isForceDeployedHere(u.getForceId()));
+            anyMatch(contract ->
+                contract.getStratconCampaignState() != null &&
+                contract.getStratconCampaignState().isForceDeployedHere(u.getForceId()));
```

Comment:
Could probably do Campaign::getActiveStratConContracts, then do the check, but that can be left for later.